index.js

```diff
@@ -26,11 +26,13 @@ app.post('/render', upload.single(`template`), async (req, res) => {
   const template = req.file;
   const originalNameWOExt = template.originalname.split(`.`).slice(0, -1).join(`.`);
   const originalFormat = template.originalname.split(`.`).reverse()[0];
+  console.log(req.body);
   let data = req.body.data;
   let options = {};
   let formatters = {};
   try {
-    options = JSON.parse(req.body.options);
+    options = {convertTo:req.body.format || originalFormat,
+    outputName:req.body.outputName || `${originalNameWOExt}.${options.convertTo}`};
```

Comment:
Why ignoring `req.body.options` ?

`convertTO` and `outputName` are already available for customization with `req.body.options` object.

Reply:
yeah I'll revert it, I was using the test.html and did not get a pdf but a odt generated.